### ISSUE TITLE
Could we get the current pull-request build status (FAILURE or SUCCESS) as variable?

### ISSUE THREAD

**Issue by ThaKing** [tomasbjerre/pull-request-notifier-for-bitbucket#274]
Could we get the current pull-request build status (FAILURE or SUCCESS) as some variable like ${PULL_REQUEST_BUILD_STATUS} ?

**Comment by ThaKing** [comment]
Yeah, it's here https://docs.atlassian.com/bitbucket-server/rest/4.0.0/bitbucket-build-rest.html#idp26432
I think it should be the last commit in current Pull Request, and the summary of all builds. Because we can get few builds result for current commit. 
If 3 builds are green => result should be green (PASSED), 
if one of 3 builds is red => result should be red (FAILED)

**Comment by tomasbjerre** [comment]
It would need to be exposed in the backend API https://docs.atlassian.com/bitbucket-server/javadoc/5.7.0/api/reference/packages.html

**Comment by ThaKing** [comment]
Ohhh, I see

**Comment by dcsobral** [comment]
Build status is attached to the commit. I looked briefly, and "summary" might be what @ThaKing is looking for. I happen to want more stuff in a request issue I'm opening, which might be found in "enrich".
